src/app/products/fixed-deposit-products/view-fixed-deposit-product/view-fixed-deposit-product.component.html

```diff
@@ -0,0 +1,421 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" fxLayout.lt-md="column">
+
+        <h2 class="mat-h2" fxFlexFill>{{ fixedDepositProductData.name }}</h2>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <hr>
+
+        <h3 class="mat-h3" fxFlexFill>Details</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill *ngIf="fixedDepositProductData.description">
+          <span fxFlex="40%">Description:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.description }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Short Name:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.shortName }}</span>
+        </div>
+
+        <hr>
+
+        <h3 class="mat-h3" fxFlexFill>Items</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Currency:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.currency.name }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Decimal Places:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.currency.decimalPlaces }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Currency in Multiples Of:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.currency.inMultiplesOf | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Minimum Deposit Amount:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.minDepositAmount | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Default Deposit Amount:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.depositAmount | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Maximum Deposit Amount:</span>
+          <span fxFlex="60%">{{ fixedDepositProductData.maxDepositAmount | number}}</span>
+        </div>
```

Comment:
Take reference from the Preview of Create Fixed Deposit Product and update. Make sure all fields that are non-mandatory are only visible if their value exists.